addons/bestja_frontend_fpbz/static/src/js/custom.js

```diff
@@ -0,0 +1,35 @@
+jQuery(window).load(function(){
+
+    $('button.choose_place').click(function(){
+        $('html, body').animate({
+        scrollTop: $("#map_of_poland_section").offset().top
+    }, 800);
+    });
+
+    var window_height = window.innerHeight;
+    $('main_top_photo_text').css({'height':window_height});
+        
+/* EVENT LISTENERS
+*----------------------------------------------------------------------------------------*/
+
+    if(window.attachEvent) {
+        window.attachEvent('onresize', function() {
+            sizeChange();
+        });
+    }
+    else if(window.addEventListener) {
+        window.addEventListener('resize', function() {
+            sizeChange();
+        }, true);
+    }
+    else {
```

Comment:
Are there any modern browsers that don’t support neither `attachEvent` nor `addEventListener`? If not, I think that we may omit that clause.